tests/js/spec/views/organizationIntegrations/integrationListDirectory.spec.jsx

```diff
@@ -0,0 +1,352 @@
+              slug: 'amazon-sqs',
+              status: 'beta',
+              type: 'data-forwarding',
+              version: '10.1.0.dev0',
+            },
+            {
+              status: 'unknown',
+              description: 'Send alerts to PagerDuty.',
+              isTestable: true,
+              isHidden: false,
+              hasConfiguration: true,
+              features: [],
+              shortName: 'PagerDuty',
+              id: 'pagerduty',
+              assets: [],
+              featureDescriptions: [],
+              name: 'PagerDuty',
+              author: {url: 'https://github.com/getsentry/sentry', name: 'Sentry Team'},
+              contexts: [],
+              doc: '',
+              resourceLinks: [
+                {
+                  url: 'https://github.com/getsentry/sentry/issues',
+                  title: 'Report Issue',
+                },
+                {
+                  url:
+                    'https://github.com/getsentry/sentry/tree/master/src/sentry_plugins',
+                  title: 'View Source',
+                },
+              ],
+              slug: 'pagerduty',
+              projectList: [
+                {
+                  projectId: 2,
+                  configured: true,
+                  enabled: true,
+                  projectSlug: 'javascript',
+                  projectPlatform: 'javascript',
+                  projectName: 'JavaScript',
+                },
+              ],
+              version: '10.1.0.dev0',
+              canDisable: true,
+              type: 'notification',
+              metadata: {},
+            },
+          ],
+        ],
+        [`/organizations/${org.slug}/repos/?status=unmigratable`, []],
+      ]);
+
+      wrapper = mountWithTheme(
+        <IntegrationListDirectory params={{orgId: org.slug}} location={{search: ''}} />,
+        routerContext
+      );
+    });
+    it('match previous snapshot', async function() {
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('shows installed integrations at the top', async function() {
+      expect(wrapper.find('SearchInput').exists()).toBeTruthy();
+      expect(wrapper.find('PanelBody').exists()).toBeTruthy();
+      expect(wrapper.find('IntegrationRow')).toHaveLength(6);
+      expect(
+        wrapper
+          .find('IntegrationRow')
+          .first()
+          .props().slug
+      ).toEqual('bitbucket');
+      expect(
+        wrapper
+          .find('IntegrationRow')
+          .at(1)
+          .props().slug
+      ).toEqual('my-headband-washer-289499');
+      expect(
+        wrapper
+          .find('IntegrationRow')
+          .at(2)
+          .props().slug
+      ).toEqual('pagerduty');
+      expect(
+        wrapper
+          .find('IntegrationRow')
+          .at(3)
+          .props().slug
+      ).toEqual('amazon-sqs');
+      expect(
+        wrapper
+          .find('IntegrationRow')
+          .at(4)
+          .props().slug
+      ).toEqual('clickup');
```

Comment:
@NisanthanNanthakumar Nit: there is probably a more concise way to do these assertions with a for loop over an array of the slugs you expect in order.